spec/ddtrace/configuration_spec.rb

```diff
@@ -259,14 +264,16 @@
           let(:statsd) { instance_double(Datadog::Statsd) }
 
           before do
-            expect(statsd).to_not receive(:close)
+            allow(statsd).to receive(:close)
 
             test_class.configure do |c|
               c.runtime_metrics.statsd = statsd
               c.diagnostics.health_metrics.statsd = statsd
             end
 
             test_class.configure { |_c| }
+
+            expect(statsd).to_not have_received(:close)
```

Comment:
I think this should also be part of the it, since this is the behavior we're after in this test (reusing and not closing).